core/server/master/src/main/java/alluxio/master/AlluxioMasterProcess.java

```diff
@@ -370,6 +375,18 @@ protected void stopServing() throws Exception {
         Thread.currentThread().interrupt();
       }
     }
+  }
+
+  /**
+   * Stops serving, trying stop RPC server and web ui server and letting {@link MetricsSystem} stop
+   * all the sinks.
+   */
+  protected void stopServing() throws Exception {
```

Comment:
Update the names correspondingly

Reply:
Done. stopCommonServing -> stopCommonServices